Makefile.Downstream.mk

```diff
@@ -65,7 +75,6 @@ bundle: kustomize operator-sdk manifests
 	cd build && echo "$$BUILD_INSTALLER_OVERLAY" > kustomization.yaml
 	cd build && $(KUSTOMIZE) edit add resource ../config/default/
 	cd build && $(KUSTOMIZE) edit add patch --path manager_downstream_tolerations.yaml --kind Deployment --name controller-manager
-	cd config/manifests/bases && $(KUSTOMIZE) edit add annotation --force 'olm.skipRange':"$(SKIP_RANGE)"
```

Comment:
instead of adding a new target can we add it here for Replace like https://github.com/red-hat-storage/ocs-client-operator/blob/7e2024d5fbb763c9fd878e4e8497c6de92f96b75/Makefile#L122C3-L123C82, trying to keep most of the downstream makefiles same